docs/dev/security-architecture.md

```diff
@@ -20,4 +20,58 @@ password, it will response authentication result with authentication Id.
  Users can defined their own JAAS Login module or write own custom authenticators as well.
   - [Changing the Default Security Settings](../user/authentication-settings-guide.md)
 
-If authentication is success, connection will be established or else will send authentication error.
\ No newline at end of file
+If authentication is success, connection will be established or else will send authentication error.
```

Comment:
shall we say "If authentication is successful"?

Reply:
Fixed in 41f8861